stable/elasticsearch/Chart.yaml

```
@@ -1,6 +1,6 @@
 name: elasticsearch
 home: https://www.elastic.co/products/elasticsearch
-version: 1.18.0
+version: 1.18.1
```

please bump to 
```suggestion
version: 1.19.0
```

adding new feature